lib/runtime.js

```diff
@@ -1,5 +1,6 @@
 "use strict";
 
+var __esModule = typeof Symbol === "function" ? Symbol.for("__esModule") : "__esModule";
 var hasOwn = Object.prototype.hasOwnProperty;
```

Comment:
The use of a symbol here avoids `Object.defineProperty` use in modern environments like Node 4+ and gives us a 30-40ms speed boost on loading (per the benchmark).